sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala

```diff
@@ -1215,20 +1215,20 @@ abstract class HiveThriftServer2TestBase extends SparkFunSuite with BeforeAndAft
     }
 
     val driverClassPath = {
-      // Writes a temporary log4j.properties and prepend it to driver classpath, so that it
+      // Writes a temporary log4j2.properties and prepend it to driver classpath, so that it
       // overrides all other potential log4j configurations contained in other dependency jar files.
       val tempLog4jConf = Utils.createTempDir().getCanonicalPath
 
       Files.write(
         """rootLogger.level = info
-          |rootLogger.appenderRef.file.ref = console
+          |rootLogger.appenderRef.stdout.ref = console
           |appender.console.type = Console
           |appender.console.name = console
           |appender.console.target = SYSTEM_ERR
           |appender.console.layout.type = PatternLayout
           |appender.console.layout.pattern = %d{yy/MM/dd HH:mm:ss} %p %c{1}: %m%n
         """.stripMargin,
-        new File(s"$tempLog4jConf/log4j.properties"),
+        new File(s"$tempLog4jConf/log4j2.properties"),
```

Comment:
`UISeleniumSuite` may need some change

Reply:
Hmm... UISeleniumSuite may need rewrite log config because it still use log4j1

https://github.com/apache/spark/blob/5eeb6ee4548699610fc39bb0f439aea5246490dd/sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/UISeleniumSuite.scala#L76-L86

Reply:
After change `HiveThriftServer2Suites` and `UISeleniumSuite`, all tests of `hive-thriftserver` module will pass with `mvn test`.
```
Run completed in 16 minutes, 24 seconds.
Total number of tests run: 552
Suites: completed 20, aborted 0
Tests: succeeded 552, failed 0, canceled 0, ignored 19, pending 0
All tests passed.
```
However, the logs in the console has increased significantly, may be we lack configuration items to replace `log4j.appender.CA.Threshold = WARN`

Reply:
I updated the console filter threshold from info to warn.